scanners/android/android.go

```diff
@@ -29,38 +31,44 @@ func (*Scanner) ExcludedScannerNames() []string {
 }
 
 // DetectPlatform ...
-func (scanner *Scanner) DetectPlatform(searchDir string) (bool, error) {
+func (scanner *Scanner) DetectPlatform(searchDir string) (b bool, err error) {
 	scanner.SearchDir = searchDir
 
-	buildGradleFiles, err := CollectRootBuildGradleFiles(searchDir)
+	scanner.ProjectRoots, err = walkMultipleFiles(searchDir, "build.gradle", "settings.gradle")
 	if err != nil {
 		return false, fmt.Errorf("failed to search for build.gradle files, error: %s", err)
 	}
 
-	return (len(buildGradleFiles) > 0), nil
+	return len(scanner.ProjectRoots) > 0, err
 }
 
 // Options ...
 func (scanner *Scanner) Options() (models.OptionModel, models.Warnings, error) {
-	return GenerateOptions(scanner.SearchDir)
+	return scanner.generateOptions(scanner.SearchDir)
 }
 
 // DefaultOptions ...
 func (*Scanner) DefaultOptions() models.OptionModel {
-	gradleFileOption := models.NewOption(GradleFileInputTitle, GradleFileInputEnvKey)
+	projectLocationOption := models.NewOption(ProjectLocationInputTitle, ProjectLocationInputEnvKey)
 
-	gradlewPthOption := models.NewOption(GradlewPathInputTitle, GradlewPathInputEnvKey)
-	gradleFileOption.AddOption("_", gradlewPthOption)
+	moduleOption := models.NewOption(ModuleInputTitle, ModuleInputEnvKey)
+	projectLocationOption.AddOption("_", moduleOption)
+
+	testVariantOption := models.NewOption(TestVariantInputTitle, TestVariantInputEnvKey)
```

Comment:
do we need to collect the variants for the default config as well? it is a not required input of the steps and we should create a less user question as possible.